src/api/AppActivity.js

```diff
@@ -115,20 +120,26 @@ class AppActivity extends MarkerBasedAPI {
         permissions: BoxItemPermission,
         successCallback: Function,
         errorCallback: ElementsErrorCallback,
+        language?: string = DEFAULT_LOCALE,
     ): void {
         const requestData = {
             item_id: id,
             item_type: 'file',
             fields: APP_ACTIVITY_FIELDS_TO_FETCH.toString(),
         };
 
+        const headers = {
```

Comment:
Can we just add this to Xhr.js for all XHRs?

Reply:
That's not a terrible idea. So you're imagining we have something like a `defautlHeaders` object that we can spread into the result going to all requests?